icons/receipt.svg

```diff
@@ -0,0 +1,15 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M4 2v20l2-1 2 1 2-1 2 1 2-1 2 1 2-1 2 1V2l-2 1-2-1-2 1-2-1-2 1-2-1-2 1-2-1Z" />
+  <path d="M16 8h-6a2 2 0 0 0-2 2v0a2 2 0 0 0 2 2h4a2 2 0 0 1 2 2v0a2 2 0 0 1-2 2H8" />
+  <path d="M12 17V7" />
```

Comment:
$ sign as a single element and optimized things a bit without changing appearance.
```suggestion
  <path d="M16 8h-6a2 2 0 0 0 0 4 2 2 0 0 0 0 0h4a2 2 0 0 1 0 4 2 2 0 0 1 0 0H8m4 1V7" />
```

Reply:
Actually, separate paths should not be merged.
I did optimize the S path, dunno why sometimes svgo adds those extra points.

Reply:
> Dunno why sometimes svgo adds those extra points.

This is the problem https://github.com/svg/svgo/pull/1754/files#r1125034286

But even with this missing, SVGO should strip them.

I submitted a fix for the stripping, but they shouldn't be added in the first place

https://github.com/svg/svgo/pull/1755